README.md

```diff
@@ -137,6 +137,13 @@ The system where FIPS compliant packages are to be installed must have Redhat's
 
 Once FIPS complaint Open-CE packages are installed in a conda environment, ensure that the conda environment contains  `openssl` and `cryptography` also from the same conda channel.
 
+### FFMPEG with reduced codecs and openssl
+#### Building packages with OpenCE ffmpeg
+
+One can build Open-CE packages with ffmpeg by providing `--build-ffmpeg` option to `open-ce build env` command. When this option is provided, Open-CE Builder builds [`envs/ffmpeg-env.yaml`](https://github.com/open-ce/open-ce/blob/main/envs/ffmpeg-env.yaml) internally and then builds the provided Open-CE environment file for which the build is requested.
+
+`--build_ffmpeg` is not supported with `--fips`. `ffmpeg` package is built by default via `openssl-env.yaml` when `--fips` is set.
```

Comment:
One can build Open-CE packages with ffmpeg in two ways:
- Provide `--build-ffmpeg` option to the `open-ce build env` command. When this option is provided, Open-CE Builder builds [`envs/ffmpeg-env.yaml`](https://github.com/open-ce/open-ce/blob/main/envs/ffmpeg-env.yaml) internally and then builds the provided Open-CE environment file for which the build is requested.
- Provide `--fips` option to the `open-ce build env` command. This will use the `openssl-env.yaml`, which includes `ffmpeg` by default.

Reply:
Updated.